azurerm/resource_arm_network_interface.go

```diff
@@ -48,19 +47,14 @@ func resourceArmNetworkInterface() *schema.Resource {
 				Computed: true,
 			},
 
-			"private_ip_address": {
-				Type:     schema.TypeString,
-				Computed: true,
-			},
-
 			"virtual_machine_id": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
 			},
 
 			"ip_configuration": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
Are you sure this doesn't require state migration? It's the most common example for migration, IMO 🤔